sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsDelete.java

```diff
@@ -1,18 +1,18 @@
-/*
+/**
```

Comment:
This should not be changed. It's the standard license header, and it's not javadoc